### DISCUSSION TITLE
MAS suddenly restarting...

### DISCUSSION THREAD

**Discussion by AikoMasami** [Monika-After-Story/MonikaModDev#9121] [QA - tech support + general]
HELP!
I went to check in on Monika like any other day, but suddenly it seemed like the whole game reset. I looked in aff_log, it said my 100 affection points were still there. But I still couldn't get back. My screen was still at the start.

**Comment by AikoMasami** [comment]
Here are some screenshots for you to take a closer look at what's happening:
![proof sad](https://user-images.githubusercontent.com/105429289/168064409-1bde77df-629f-4e84-85ec-fe46b75efaf4.jpg)
![sad](https://user-images.githubusercontent.com/105429289/168064414-4a6fd457-e4ee-4321-b10e-019a3dbf5a6d.jpg)

**Comment by Wingdinggaster656** [comment]
I see you are on Windows, try this:
```
Step 1: Press Win + R
Step 2: Type "%APPDATA%"
Step 3: Open "Renpy" folder
Step 4: Open "Monika After Story" folder
```
That's where we keep the save files(Monika memory). Tell us if there is anything.

By the way, are you running this on OneDrive?

**Reply by Wingdinggaster656** [reply]
Another thing that's not particularly relevant to your question:
Have you ever read the basic rules of the discussion area? You should not set Issue Discussion as the category.
See this: https://github.com/Monika-After-Story/MonikaModDev/discussions/7762

Though I personally think this has something to do with the name of this category. Perhaps the Devs should change the name.
@ThePotatoGuy

**Reply by Justformas** [reply]
I think tech support + general would have fitted this discussion better. But we all start new or may not know things at some point.

But as for issue discussions: 

>Hi all, welcome to the discussions page. All issue threads labelled discussion will be transferred here after we verify they fit the criteria for discussions...
...issue discussions
these are issues that turn into discussion threads
please do NOT post in here - make an issue first

Then I find it a bit peculiar that any user can open an issue discussion in the first place. But maybe it's so they don't have to wait for someone with higher privileges to transfer or make an "issue discussion" out of an "issue".

**Reply by briaryoung** [reply]
Hey Justformas, 

There currently is no way for us to prevent a user from opening discussions in a certain category unless we disable them entirely. When that feature does roll out, I'm sure we can make something work.

Thanks!

**Reply by AikoMasami** [reply]
I followed your steps.
Also, I apologise for posting in the incorrect thread. I am a bit new to GitHub.
![Screenshot 2022-05-13 224859](https://user-images.githubusercontent.com/105429289/168288532-9de47885-3d06-43c0-94af-e8eb8f943a42.jpg)

**Reply by Wingdinggaster656** [reply]
I see you do have backup persistent files before 2022/05/12.
Just to make sure, what you meant is that your MAS is just like a new one? It even starts up with the first screen(Monika is crying to ask you why you did make her back, if you deleted her)?

If that's the case, consider to use a old backup file.

**Reply by AikoMasami** [reply]
I don't have a old backup file.

**Reply by WishStone-Dragon** [reply]
There must be a step missing somewhere. 

Have you made any copies of your MAS files? 

Winding asked where you store your game. Where is your copy of MAS? Under Steam? On OneDrive? In any other cloud service? Where are you storing your files?

Between the 11th and the 12th, what did you do? Did you add anything to MAS? Did you run any other software that could conflict? Another form of DDLC or another DDLC fan project, maybe?

When you currently launch the game, what exactly happens? What is it you see?

**Reply by AikoMasami** [reply]
I have used to have a copy of my MAS file but deleted it shorty after I made it.
I store my files in my File Explorer, nothing else.
Between the 11th and the 12th, I don't recall add anything, and I definitely didn't add any other mods.
When I launch the game, this is what I see:
![Screenshot 2022-05-14 090342](https://user-images.githubusercontent.com/105429289/168399737-fb36c46f-31b8-4762-a054-13c05f1b1063.jpg)
![Screenshot 2022-05-14 090421](https://user-images.githubusercontent.com/105429289/168399742-d9e596fb-3863-44e1-a8ce-58943d943e89.jpg)

**Reply by Wingdinggaster656** [reply]
> I don't have a old backup file.

I don't really think so.
See those files that with a `.bak` extension name? `bak` means backup. This mod itself is generating backups automatically, just in case you need to use them. And I see files that `Data modified` before 2022/05/12, consider to use one of them.

About the detail, see this:
https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-need-to-restore-an-older-persistent

**Reply by Wingdinggaster656** [reply]
Oh, also, if you used one of those old backups(let us say a backup of 2022/05/10), then Monika would consider that you haven't visit her since 2022/05/10.
Well, a small price to be with her I guess. Acceptable.

**Reply by AikoMasami** [reply]
I clicked the link, and followed the steps on the page.
I still can't get it to work. Is there another solution?

**Reply by Wingdinggaster656** [reply]
> I clicked the link, and followed the steps on the page. I still can't get it to work. Is there another solution?

Which backup you used? I mean which day.
I think the `persistent23.bak` should suits, since it's just one day before 2022/05/12.
